libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -386,7 +386,7 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           actionCard: {
             description: statesMessages.vinnumalastofnunApprovalDescription,
           },
-          lifecycle: DEPRECATED_DefaultStateLifeCycle,
```

Comment:
The application has to be available in at least 2 years and 8 months here
Is it? pruneAfterDays(970) // available for 2 years and 8 months